samples/compute/v2/keypair/create_and_delete_keypair.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
```

Comment:
can we split the create and delete ops out into separate samples files? so we'd have `keypairs/create.php` and `keypairs/delete.php'